### PR TITLE
Cleanup coding standards in Gdn_Format

### PR DESCRIPTION
**Pull request by linc**
Adding 2 deprecated statements are the only functional changes.

### CODE REVIEW

File: library/core/class.format.php [hunk 1]
```diff
@@ -1771,7 +1780,7 @@ public static function raw($Mixed) {
     }
 
     /**
-     * Takes an object and convert's it's properties => values to an associative
+     * Takes an object and converts its properties => values to an associative
```
Comment (beckyvb) [inline library/core/class.format.php]: nope... aaand nope :)

Comment (linc) [inline library/core/class.format.php]: huh?

File: library/core/class.format.php [hunk 2]
```diff
@@ -2021,6 +2031,12 @@ public static function toTimezone($Timestamp) {
         return $Timestamp;
     }
 
+    /**
+     *
+     *
+     * @param $timespan
```
Comment (initvector) [inline library/core/class.format.php]: What're the types for `$timespan`?

File: library/core/class.format.php [hunk 3]
```diff
@@ -2076,10 +2092,12 @@ public static function unserialize($SerializedString) {
         if (is_string($SerializedString)) {
             if (substr_compare('a:', $SerializedString, 0, 2) === 0 || substr_compare('O:', $SerializedString, 0, 2) === 0) {
                 $Result = unserialize($SerializedString);
-            } elseif (substr_compare('obj:', $SerializedString, 0, 4) === 0)
+            } elseif (substr_compare('obj:', $SerializedString, 0, 4) === 0) {
                 $Result = json_decode(substr($SerializedString, 4), false);
-            elseif (substr_compare('arr:', $SerializedString, 0, 4) === 0)
+            }
+            elseif (substr_compare('arr:', $SerializedString, 0, 4) === 0) {
```
Comment (initvector) [inline library/core/class.format.php]:
```php
} elseif (substr_compare('arr:', $SerializedString, 0, 4) === 0) {
```

File: library/core/class.format.php (hunk 2 above)
Comment (linc) [inline library/core/class.format.php]: Is "I don't care" an acceptable answer? :P

**Comment by linc** [comment]
Fixed

**Pull request merged**